store/src/java/com/zimbra/cs/mailbox/ContactAutoComplete.java

```diff
@@ -764,14 +764,14 @@ public void addMatchedContacts(String query, Map<String, ? extends Object> attrs
         Mailbox mbox = MailboxManager.getInstance().getMailboxByAccountId(getRequestedAcctId());
         if (folderIDs == null) {
             for (Folder folder : mbox.getFolderList(octxt, SortBy.NONE)) {
-                if (folder.getDefaultView() != MailItem.Type.CONTACT || folder.inTrash()) {
-                    continue;
-                } else if (folder instanceof Mountpoint) {
+                if (folder instanceof Mountpoint) {
```

Comment:
looks like changes from some other ticket

Reply:
Yes Saket, Sorry, I did it mistakenly. Reverted.